config.json

```diff
@@ -3,6 +3,10 @@
     "guild": 586127025228742656,
     "log_channel": 646748400678862848,
     "modules": {
+		"stack_overflow":{
+			"googlesearch_engine_id": "010164265242121245845%3Afxovbkirsvh",
+			"googlesearch_key": "AIzaSyAHcDOWOnhYVldRHkaTorTpCvBaafswU-w"
```

Comment:
is this a secret ?

Reply:
whoops that's a good point. I'll have to store this info in a file that isn't public to the repo